secret/builtin.go

```diff
@@ -34,3 +35,17 @@ var builtinRules = []Rule{
 		Regex:    regexp.MustCompile(`gho_[0-9a-zA-Z]{36}`),
 	},
 }
+
+var builtinGlobalAllowList = AllowList{
+	Title: "global allow list",
+	Regexes: []*regexp.Regexp{
+		regexp.MustCompile(`219-09-9999`),
+		regexp.MustCompile(`078-05-1120`),
```

Comment:
I looked it up, all three of them refer to American Social Security Numbers, they are common SSN examples.
We can make two of them use string comparison, or combine them all into one regex. 
Third option is just to remove them because I don't think they are common enough.
What do you think?

Reply:
We can remove them first. If we see many complaints about them, we can add these values then.

Reply:
👍🏼